portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,28 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.List;
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Kevin!</h1>");
+    List<String> messages = new ArrayList<String>();
+    messages.add("Hello");
+    messages.add("Good day");
+    messages.add("Thank you");
```

Comment:
A few things on collections:

1. Let's prefer to use ImmutableLists (immutable means they can't be modified after constructing) when it makes sense to. In general, immutable objects are preferred when possible because they're generally safer. You can use the com.google.common.collect version: https://guava.dev/releases/23.0/api/docs/com/google/common/collect/ImmutableList.html.
In general, let's try to use this library for all container-like data structures (lists, maps, etc). Guava in general (Google's public version of Java that contains extra libraries like this one) should be utilized when possible. See https://guava.dev/ to learn more.

2. Let's make this a constant, class variable instead of defining it inside of this function. So at the top of this class (after the definition but before this method), you can declare a constant and use the ImmutableList.of() method. So it should look something like:
private static final ImmutableList MESSAGES = ImmutableList.of("Hello ...", "Good day...", ....);

The reason is that it's good to separate the logic from the constants that don't change. If you ever just need to add another greeting message, you don't need to sift through the logic code to find this variable. Likewise, it makes the actual business logic of the doGet method a lot cleaner and concise.

Reply:
Oh btw, I think I missed some bracket formatting so in my example declaration in 2, it should be:

private static final ImmutableList<String> MESSAGES...

Still getting used to GitHub :\